modules/apigee-x-core/variables.tf

```diff
@@ -36,9 +36,16 @@ variable "ax_region" {
   type        = string
 }
 
+variable "disable_vpc_peering" {
+  description = "Flag that specifies whether the VPC Peering through Private Google Access should be disabled between the consumer network and Apigee."
```

Comment:
Shouldn't this say through "Service Networking"? PGA is something different.

Reply:
My bad. It should be "Private Service Access". Mentioning SN as well.

https://cloud.google.com/vpc/docs/configure-private-services-access